src/vt_flash/nvt.jl

```diff
@@ -0,0 +1,222 @@
+"""
+    helmholtz(mix, nmol, volume, RT[; buf])
+
+Dimensionless Helmholtz free energy [Joules / RT] of `mix`ture
+at `nmol`, `volume` and `RT`.
+
+To reduce intermediate allocations, use `buf`, see [`thermo_buffer`](@ref).
+"""
+function helmholtz(
+    mix::BrusilovskyEoSMixture,
+    nmol::AbstractVector,
+    volume::Real,
```

Comment:
Those file dedicated to "plain" nvt calculations, most of calculations based on `AbstractVTFlashState` uses function from this script and then scales gradient and hessian.